src/datetime.rs

```diff
@@ -171,6 +171,16 @@ impl crate::DateTime {
         self.to_chrono()
```

Comment:
Could we make `to_rfc3339` public as part of this work? I know it was omitted from the ticket description, but I think that was accidental. As part of that I guess we'll want to rename this to `to_rfc3339_string` to match the `parse_rfc3339_str`.